### DISCUSSION TITLE
Password sent in clear text when not using HTTPS

### DISCUSSION THREAD

**Discussion by MatsAndT** [coollabsio/coolify#3967] [Q&A]
When I set up Coolify for the first time, I was connected to it directly via the IP (HTTP) address in the browser. When I then set up the user, we noticed that the password to the admin account was sent in unencrypted text. 
Tried to log in again via the IP (HTTP) address, the password was unencrypted.

Does anyone know if this is the intention?

**Comment by centralhardware** [comment]
password hashing must be done on the server side, so you should always use https
